pkg/sink/codec/debezium/codec.go

```diff
@@ -529,7 +714,45 @@ func (c *dbzCodec) writeBinaryField(writer *util.JSONWriter, fieldName string, v
 	writer.WriteBase64StringField(fieldName, value)
 }
 
-func (c *dbzCodec) EncodeRowChangedEvent(
+// EncodeKey encode RowChangedEvent into key message
+func (c *dbzCodec) EncodeKey(
+	e *model.RowChangedEvent,
+	dest io.Writer,
+) error {
+	// schema field describes the structure of the primary key, or the unique key if the table does not have a primary key, for the table that was changed.
+	// see https://debezium.io/documentation/reference/stable/connectors/mysql.html#mysql-events
+	cols, colInfos := e.HandleKeyColInfos()
+	jWriter := util.BorrowJSONWriter(dest)
+	defer util.ReturnJSONWriter(jWriter)
+
+	var err error
+	jWriter.WriteObject(func() {
+		jWriter.WriteObjectField("payload", func() {
+			for i, col := range cols {
+				colx := model.Column2ColumnDataX(col)
```

Comment:
Could we use GetColumnDataX instead of  Column2ColumnDataX as writeDebeziumFieldValues.

Reply:
GetColumnDataX has existed. https://github.com/pingcap/tiflow/blob/45b439b210688a28afc46bcb47fb24c81b02d6c7/cdc/model/sink.go#L1309